src/comparator/wrappers.py

```diff
@@ -97,7 +97,7 @@ class Enum:
     source_code_locations: Dict[Tuple[int, ...], descriptor_pb2.SourceCodeInfo.Location]
     path: Tuple[int, ...]
     full_name: str
-    nested_path: Sequence[str]
+    nested_path: List[str]
```

Comment:
It seems `Sequence` is preferred for arguments than `List`.

https://docs.python.org/3/library/typing.html#typing.List
```
class typing.List(list, MutableSequence[T])
Generic version of list. Useful for annotating return types. To annotate arguments it is preferred to use an abstract collection type such as Sequence or Iterable.
```

Reply:
Right, but when I try to append some elements to the sequence by `nested_path + ["1", "2"]`, the typing complains about `sequence[] + list[]`. Or I can use "extend" for each sequence addition.

Reply:
I see.